org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/discovery/ZigBeeDiscoveryService.java

```diff
@@ -227,4 +249,20 @@ public void run() {
 
         scheduler.schedule(pollingRunnable, 10, TimeUnit.MILLISECONDS);
     }
+
+    private void parseAndSetCreateResultsOnlyDuringActiveScansConfigProperty(Map<String, Object> properties) {
```

Comment:
Isn't it possible to use a shorter method name?

Reply:
Yes sure, I'll have a look probably tomorrow.